From 28d6dc78d20ad993c897528f886e2993fbc091d5 Mon Sep 17 00:00:00 2001
Message-Id: <28d6dc78d20ad993c897528f886e2993fbc091d5.1664436482.git.geliang.tang@suse.com>
From: Geliang Tang <geliang.tang@suse.com>
Date: Thu, 29 Sep 2022 15:27:59 +0800
Subject: [PATCH] Revert "Linux 5.10.146"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 62aea694445d5fc0f51b45afe8003ff3b7431141.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 26a871eebe92..76c85e40beea 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 146
+SUBLEVEL = 145
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

